CMakeLists.txt

```diff
@@ -312,6 +312,12 @@ if(WIN32)
 	add_definitions(-D_CRT_SECURE_NO_DEPRECATE)
 endif(WIN32)
 
+# Add an option to build relocatable binaries on Linux
+# The Sys folder will need to be copied to the Binaries folder.
+if(UNIX)
+	add_definitions(-DLINUX_LOCAL_DEV=0)
```

Comment:
Did you intend for this to be enabled by default? Would a CMake option be a good idea here?